src/library_dylink.js

```diff
@@ -58,7 +58,12 @@ var LibraryDylink = {
       '__cpp_exception',
       '__wasm_apply_data_relocs',
       '__dso_handle',
-      '__set_stack_limits'
+      '__tls_size',
```

Comment:
Is this part actually NFC?

Reply:
Yes, I guess technically its not, but the main thing this does is prevent a lot of log messages saying:

```UpdateGOT: EXISTING SYMBOL:  <symbol_name>```

when trying to add these symbols to the GOT.